src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -984,9 +979,3 @@ def _get_agent_token(self, name: str) -> Dict[str, str]:
         token_name = f"KBApp_{name}"
         token_name = token_name[: self.__MAX_TOKEN_NAME_LEN]
         return auth.get_agent_token(auth.get_auth_token(), token_name=token_name)
-
-    def register_new_job(self, job: Job) -> None:
-        JobManager().register_new_job(job, refresh=False)
-        with exc_to_msg("appmanager"):
-            JobComm().lookup_job_state(job.job_id)
-            JobComm().start_job_status_loop()
```

Comment:
These had no obvious function, unless generating unnecessary messages is considered a useful function.